### ISSUE TITLE
[Bug Report]: About external config

### ISSUE THREAD

**Issue by deyihu** [farm-fe/farm#1957]
### Steps to reproduce

```json
  "devDependencies": {
    "@farmfe/cli": "^1.0.4",
    "@farmfe/core": "^1.4.2",
    "@vitejs/plugin-vue": "^5.0.4",
    "core-js": "^3.30.1"
  }
```
```js config file
import { defineConfig } from '@farmfe/core';


export default defineConfig({
  compilation: {
    external: [
      { jquery: "$" },
    ],
    output: {
      targetEnv: "browser",
      format: "esm"
    },
  },

});


```

### What is actually happening?

I don't know if it's my question about `external` configuration or a bug

Using a simple jQuery error(by umd script)

![G9F4)H%GS{3L65D%T ILU}T](https://github.com/user-attachments/assets/3c1b8b61-ed53-4fbc-9abc-983745c123d9)

I looked at the translated code and didn't find anything  about `module.f` fucntion 

![H2F JM RQ@4L(3SX39N2P67](https://github.com/user-attachments/assets/ba0451c8-7a20-4375-a675-6fffdb938db5)

Here is a simple reproduction of the engineering code

[jquery-external.zip](https://github.com/user-attachments/files/17856142/jquery-external.zip)

### System Info

```Shell
windows
```

**Comment by wre232114** [comment]
I'll take a look

**Comment by wre232114** [comment]
fixed in @farmfe/core@1.4.3
